docker-compose.yml.example

```diff
@@ -0,0 +1,84 @@
+version: '3.2'
+
+services:
+  nginx-proxy:
+    image: jwilder/nginx-proxy:alpine
+    ports:
+      - "80:80"
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+
+
+  proxy:
```

Comment:
Let's call this `sso-proxy` for consistency!

Reply:
good point, fixed